packages/loader/container-loader/src/container.ts

```diff
@@ -568,6 +568,8 @@ export class Container
 	private setAutoReconnectTime = performance.now();
 
 	private noopHeuristic: NoopHeuristic | undefined;
+	// Number of no-ops sent via this client in the current active connection
+	private noopCount: number = 0;
```

Comment:
This should indicate that it's per connection so that it is not mistaken for the total noops or something. Something like `noOpCountPerConnection`?